lms/djangoapps/certificates/api.py

```diff
@@ -27,7 +27,7 @@
     CertificateTemplate,
 )
 from certificates.queue import XQueueCertInterface
-
+from branding import api as branding_api, get_logo_url
```

Comment:
@mattdrayer `get_logo_url` is defined in `branding/__init__.py`, that is why I have to import it explicitly here.